modules/configuring-huge-pages.adoc

```diff
@@ -10,43 +14,66 @@ Node Tuning Operator to allocate huge pages on a specific node.
 tuned profile, which describes how many huge pages should be allocated:
 +
 ----
-$ oc label node <node-using-hugepages> hugepages=true
+$ oc label node <node_using_hugepages> hugepages=true
 ----
 
-. Create a tuned profile for this specific node and allocate some huge pages. To
-associate the profile to the correct node, edit the tuned resource and add a
-`profile` and `match` section:
+. Copy the default tuned profile and customize it to create a tuned profile
+allocating huge pages for your labeled node.
+
+.. Export the default tuned profile and save it to a file (`tuned.yaml`):
 +
 ----
-$ oc edit tuned
+$ oc get tuned --export -o yaml > tuned.yaml
 ----
 
-.. In the `profile` section, add the following tuned profile for allocating huge
-pages:
+.. To rename the profile to `hugepages`, edit the `tuned.yaml` file and update
+the `metadata.name` parameter:
 +
 ----
-- data: |
-    [main]
-    summary=Configuration for hugepages
-    include=openshift-node
-    [vm]
-    transparent_hugepages=never
-    [sysctl]
-    vm.nr_hugepages=1024
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: hugepages <1>
+  namespace: openshift-cluster-node-tuning-operator
 ----
+<1> Rename the profile.
 
-.. To apply the profile to the correct node, add the following to the `match`
+.. To add a tuned profile for allocating huge pages and associate the profile to
+the correct node, add the following `profile` and `match` sections to the `spec`
 section:
 +
 ----
- - match:
-   - label: hugepages
-   priority: 30
-   profile: hugepages
+spec:
+  profile: <1>
+  - data: |
+      [main]
+      summary=Configuration for hugepages
+      include=openshift-node
+
+      [vm]
+      transparent_hugepages=never
+
+      [sysctl]
+      vm.nr_hugepages=1024
+    name: node-hugepages
+  recommend:
+  - match: <2>
+    - label: hugepages
+    priority: 30
+    profile: node-hugepages
+----
+<1> Add a `profile` section.
```

Comment:
Are there any customizations that the user should consider in these two sections, or must they use these exact settings?

Reply:
^ @skordas 	@jmencak 	?

Reply:
The CR is a good start -- it works now...  I'd leave this as is.  As for customizations, perhaps the number of "persistent" huge pages in the kernel's huge page pool (nr_hugepages) and profile priority should be considered as "customizations".